frontend/src/components/user/SignUpForm.js

```diff
@@ -32,9 +33,14 @@ const SignUpForm = ({ navigate }) => {
     setPassword(event.target.value)
   }
 
+  const handleNameChange = (event) => {
+    setName(event.target.value)
+  }
+
 
     return (
       <form onSubmit={handleSubmit}>
+          <input placeholder="Full Name" id="name" type='name' value={ name } onChange={handleNameChange} />
```

Comment:
So are we going with Full Name instead of First Name & Last Name now?
I am good with the decision. I just want to confirm it. Thank you!

Reply:
This was my observation also. It might be better to have separate first and last name fields so that the first name can be used for any system greetings (''Hi (x)")? If we're going with a merged name approach this might as well be stated as 'username' in my opinion. I just don't think I've ever seen a site that requires full name as a single field?

That said, this is a minor stylistic point with future functionality in mind, so happy to go with this if that's the group consensus.